### PR TITLE
Expanded the bug reporting guidelines

### PR DESCRIPTION
**Pull request by Mailaender**
A humble wish.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 * Have you read the [FAQ](https://github.com/OpenRA/OpenRA/wiki/FAQ)?
 * Add the appropriate log files on crashes.
 * Please be specific on how to reproduce the problem.
+* Do not submit duplicate issues. Search first.
+* Avoid including few requests/bugs into a single issue.
```
Comment (penev92) [inline CONTRIBUTING.md]: "few" -> "several"

**Comment by penev92** [comment]
:+1: although I doubt most people read those. :/

**Comment by Phrohdoh** [comment]
s/few/multiple

**Comment by clemty** [comment]
Perhaps tell people to say which version they are using? (to avoid reports for long-fixed bugs in bleed because people use the last release)

**Comment by Mailaender** [comment]
Updated.

> Perhaps tell people to say which version they are using?

The situation of not mentioning the version has improved since https://github.com/OpenRA/OpenRA/pull/3938, but nonetheless that is important and often forgotten and a problem when no logs are provided. Added.

**Comment by obrakmann** [comment]
:+1:

**Pull request merged**